crates/aranya-daemon/src/daemon.rs

```diff
@@ -330,9 +333,13 @@ impl Daemon {
         eng: CE,
         store: AranyaStore<KS>,
         pk: &PublicKeys<CS>,
-        external_sync_addr: Addr,
-        psk_store: Arc<PskStore>,
-        active_team_rx: Receiver<TeamId>,
+        SyncParams {
+            psk_store,
+            active_team_rx,
+            external_sync_addr,
```

Comment:
nit: since this is the address the server binds to:
```suggestion
            server_addr,
```
`_sync_` isn't needed in the name since that's implied by the field being inside the `SyncParams` struct.